index.html

```diff
@@ -1677,9 +1677,19 @@ <h2>
             alphabetic code (i.e., the numeric codes are not supported). Their
             canonical form is upper case. However, the set of combinations of
             currency code for which localized currency symbols are available is
-            implementation dependent. Where a localized currency symbol is not
-            available, a user agent SHOULD use U+00A4 (¤) for formatting. User
-            agents MAY format the display of the
+            implementation dependent.
+          </p>
+          <p>
```

Comment:
Edited down and re-ordered slightly: 

 <p>When displaying a monetary value, it is
            RECOMMENDED that user agents display the currency code and OPTIONAL for user agents to display a currency symbol. Note that            currency symbols can be ambiguous due to use across a number of
            different currencies (e.g., "$" could mean any of USD, AUD, NZD,
            CAD, and so on.). User agent that display currency symbols MAY use U+00A4 (¤) when a currency symbol is not  available.
</p>
<p>User agents MAY format the display of the
            {{PaymentCurrencyAmount/currency}} member to adhere to OS
            conventions (e.g., for localization purposes).
          </p>

Reply:
Ok, looks good! let me integrate that. The "and OPTIONAL" is a bit awkward, so I might rephrase that a tiny bit. But apart from that, all seems good.

Reply:
Cool! If you want to update the PR I can wrap up the review rapidly. Thanks again!

Reply:
on it!